modules/EventEmitter.js

```diff
@@ -1100,9 +1082,7 @@ class EventEmitter {
 
         // async
         this._on('kad-send-encrypted-key', async (request, response) => {
-            await transport.sendResponse(response, {
-                status: 'OK',
-            });
+            await transport.sendResponse(response, []);
```

Comment:
why don't we send this response in the kademlia file, like we did for other routes?

Reply:
Not used at the moment. In general, all others should be changed in future.